modules/tinymce/src/themes/silver/main/ts/ui/menus/item/structure/ItemSlices.ts

```diff
@@ -88,6 +89,38 @@ const renderDownwardsCaret = (icons: IconProvider): AlloySpec => ({
   }
 });
 
+const renderContainer = (components: Array<AlloySpec>, direction: string): AlloySpec => ({
+  dom: {
+    tag: 'div',
+    classes: [ direction === 'vertical' ? ItemClasses.containerColumnClass : ItemClasses.containerRowClass ]
+  },
+  components
+});
+
+const renderImage = (src: string, width: Optional<string>, height: Optional<string>, alt: Optional<string>): AlloySpec => ({
+  dom: {
+    tag: 'img',
+    classes: [ ItemClasses.imageClass ],
+    styles: {
+      'vertical-align': 'top'
```

Comment:
Why is this here? This doesn't seem like a common thing we'd want and this should probably instead be in the css.

Reply:
Correct, commited this by mistake when experimenting.